nodebb-theme-mytheme/templates/quickreply.tpl

```diff
@@ -0,0 +1,29 @@
+{{{ if privileges.topics:reply }}}
+<div component="topic/quickreply/container" class="quick-reply d-flex gap-3 mb-4">
+	<div class="icon hidden-xs">
+		<a class="d-inline-block position-relative" href="{{{ if loggedInUser.userslug }}}{config.relative_path}/user/{loggedInUser.userslug}{{{ else }}}#{{{ end }}}">
```

Comment:
are these changes from a different commit?

Reply:
yep they're not from my commit.